src/javascript/app-controller.js

```diff
@@ -0,0 +1,15 @@
+import {
+  render,
+  checkForClick,
+  changeTemp,
+} from './dom-work';
+
+function start() {
```

Comment:
**REQUIRED**
We should strictly use ES6 syntax. Let's replace function with syntax with and arrow function. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from Pathwright)